pkg/controllers/networkinfo/networkinfo_controller.go

```diff
@@ -146,6 +139,13 @@ func (r *NetworkInfoReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 			log.Error(err, "failed to get NSX VPC ConnectivityProfile object", "vpcConnectivityProfileName", vpcConnectivityProfileName)
 			return common.ResultRequeue, err
 		}
+		if len(vpcConnectivityProfile.ExternalIpBlocks) == 0 {
+			if ncName == commonservice.SystemVPCNetworkConfigurationName {
+				setVPCNetworkConfigurationStatusWithNoExternalIPBlock(ctx, r.Client, vpcNetworkConfiguration)
+				log.Error(err, "there is no ExternalIPBlock in VPC ConnectivityProfile", "VPC", req.NamespacedName)
+				return common.ResultRequeue, fmt.Errorf("no ExternalIPBlock in VPC ConnectivityProfile, VPC: %s", req.NamespacedName)
```

Comment:
Do not return, just set CR status and continue creating VPC.
And also need to reset the status to True after external IPBlocks are configured later

Reply:
Updated.